index.html

```diff
@@ -228,36 +221,67 @@ <h2 class="about-head">About me</h2>
           </ul>
         </div>
       </section>
-  <section class="contact">
-  <form action="https://formspree.io/f/xlezlkyw" method="post">
-          <p class="form-detail">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-             <input type="text" placeholder="Full Name" maxlength="30" required>
-             <input type="email" placeholder="Email address" required>
-             <textarea name="msgt1" id="t1" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+      <section class="contact">
+        <div class="form-detail">
           <p>
-            <button class="button" type="submit">Get in touch</button>
-           </p>
-  </form>
-  <footer class="social-footer-social">
-  <div class="social-media-icon1">
-    <a href="#"  class="social-link">
-      <img class="social-icon" src="./icons/github.png" alt="GitHub"/>
-    </a>
-    <a href="#"  class="social-link">
-      <img class="social-icon" src="./icons/Linkedin.png" alt="LinkedIn"/>
-    </a>
-    <a href="#"  class="social-link">
-      <img class="social-icon" src="./icons/angel.png" alt="Angel"/>
-    </a>
-    <a href="#"  class="social-link">
-      <img class="social-icon" src="./icons/twitter.png" alt="Twitter"/>
-    </a>
-    <a href="#"  class="social-link">
-    <img class="social-icon" src="./icons/medium.png" alt="Medium"/>
-    </a>
-  </div>
-  </footer>
-</section>
-</main>
-</body>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </p>
+        </div>
+        <div class="contact-form">
+          <form
+            action="https://formspree.io/f/xlezlkyw"
+            method="post"
+            class="contact-us"
+          >
+            <input
+              type="text"
+              placeholder="Full Name"
+              maxlength="30"
+              required
+            />
+            <input type="email" placeholder="Email address" required />
+            <textarea
+              name="msgt1"
+              id="t1"
+              cols="30"
+              rows="10"
+              maxlength="500"
+              placeholder="Enter text here"
+              required
+            ></textarea>
+            <p>
+              <button class="button" type="submit">Get in touch</button>
+            </p>
+          </form>
```

Comment:
- [ ] Kindly increase the width of the form elements to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/160218705-a7757900-365d-4f58-a0ba-f96d947085ed.png) | ![image](https://user-images.githubusercontent.com/24703171/160218715-e6cb0a26-6398-472a-875a-5e146cf75415.png) |

Reply:
@gemmen29 Please how am i supposed to maintain the same width if you ask i use a template of 768px as against the figma template of 1440px?

Reply:
@Adanzeakonobi, By using `%` in style you can build a more responsive website.